src/components/Forms/Register-Customer.js

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from 'react';
+
+class RegisterCustomer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fName: '',
+      lName: '',
+      email: '',
+      phone: '',
+      zipcode: '',
+      address: '',
+      city: '',
+      state: '',
+      country: '',
+      photo_url: '',
+      //   password: '',
+    };
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+    //   testing the handlesubmit
+    const data = this.state;
+    console.log('Final Data', data);
+  };
+
+  handleInputChange = event => {
+    event.preventDefault();
+    //   getting name of input and value
+    // console.log('handleINputChange Name', event.target.name);
+    //   console.log('handleINputChange Value', event.target.value);
+    this.setState({
+      // seting key to key-value pair
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  // must be async in order to await key function
+  async postDataGroomers() {
+    try {
+      let result = await fetch(
+        'https://labspt12-express-groomer-a-api.herokuapp.com/groomers',
+        {
+          method: 'post',
+          mode: 'no-cors', // no-cors to eliminate some errors
+          headers: {
+            Accept: 'application/json',
+            'Content-type': 'application/json',
+          },
+          // stringify the json obj payload
+          body: JSON.stringify({
+            fName: this.state.fName,
+            lName: this.state.lName,
+            email: this.state.email,
+            phone: this.state.phone,
+            address: this.state.address,
+            city: this.state.city,
+            state: this.state.state,
+            country: this.state.country,
+            zipcode: this.state.zip,
+            photo_url: this.state.photo_url,
+          }),
+        }
+      );
+      console.log('RESULT', result);
+    } catch (e) {
+      console.log('REGISTER ERROR: ', e);
+    }
+  }
+
+  componentDidMount() {
+    this.setState({
+      fName: this.state.fName,
+      lName: this.state.lName,
+      email: this.state.email,
+      phone: this.state.phone,
+      address: this.state.address,
+      city: this.state.city,
+      state: this.state.state,
+      country: this.state.country,
+      zipcode: this.state.zip,
+      photo_url: this.state.photo_url,
+      //   password: this.state.password,
+    });
+  }
```

Comment:
I think this code might be redundant. Your state is getting initialized on line 6, so there isn't any need to change their values when the component mounts.